@W-23154741 - Devops promote commands (#510)
salesforcecli/plugin-devops-center: messages/devops.stage.promote.md

# summary

Promote approved work items to a target pipeline stage.

# description

> Promotes all approved work items in a project to the specified target stage. Specify the ID of the target pipeline stage to promote to.

Reviewer: Don't simply repeat the command summary in the command description.   The part about specifying the ID isn't really that interesting, you show how to do that in the examples.  So I would remove this description if there's nothing more to say.

# flags.target-stage-id.summary

ID of the pipeline stage to promote work items to.

# examples

> - Promote all approved work items to a stage:

Reviewer: ```suggestion
- Promote all approved work items to a stage with the specific ID; use the org with alias "my-devops-org":
```

      <%= config.bin %> <%= command.id %> --target-org my-devops-org --target-stage-id 05S000000000001

> - Promote and deploy all metadata, running all local tests:

Reviewer: ```suggestion
- Promote and deploy all metadata, and run all local tests:
```

      <%= config.bin %> <%= command.id %> --target-org my-devops-org --target-stage-id 05S000000000001 --deploy-all --test-level RunLocalTests

# error.NoWorkItems

> No work items found to promote from the source stage. Ensure there are approved work items before promoting.

Reviewer: ```suggestion
No work items found to promote from the source stage. Make sure there are approved work items before promoting.
```
